pkg/manager/namespace_test.go

```diff
@@ -90,24 +89,21 @@ var _ = Describe("ConfigureWatchNamespaces", func() {
 			By("configuring WATCH_NAMESPACE with empty string")
 			Expect(os.Setenv(WatchNamespaceEnvVar, ""))
 
-			By("configuring WATCH_NAMESPACE with the namespaces of the watched pods")
-			ConfigureWatchNamespaces(&opts, log)
```

Comment:
Was this dropped intentionally? Seems like we need to call it to make sure it correctly configures the options based on the env.

Reply:
Wasn't intentional

Reply:
https://github.com/operator-framework/helm-operator-plugins/pull/270